lib/driver.rb

```diff
@@ -43,5 +43,34 @@ def add_trip(trip)
 
       @trips << trip
     end
+
+    def calculate_driver_revenue
+      fee = 1.65
+      driver_takehome = 0.8
```

Comment:
In general, you don't need to include the name of the class in instance methods for that class. Something like `calculate_revenue` or `total_revenue` would be a little more concise.